e2e/bzlmod/MODULE.bazel

```diff
@@ -0,0 +1,28 @@
+module(
+    name = "e2e_bzlmod",
+    version = "0.0.0",
+    compatibility_level = 1,
+)
+
+bazel_dep(name = "aspect_rules_ts", version = "0.0.0", dev_dependency = True)
+bazel_dep(name = "bazel_skylib", version = "1.1.1", dev_dependency = True)
+
+local_path_override(
+    module_name = "aspect_rules_ts",
+    path = "../..",
+)
+
+# TODO: remove this after rules_js 0.10.0 pushed to aspect bcr
+local_path_override(
+    module_name = "aspect_rules_js",
+    path = "../../../rules_js",
+)
+
+rules_ts_ext = use_extension("@aspect_rules_ts//ts:extensions.bzl", "ext", dev_dependency = True)
+
+rules_ts_ext.npm_typescript(
+    name = "npm_typescript",
+    ts_version = "4.6.3",
+)
```

Comment:
Any opinions on the naming for the extension/method?

Reply:
Something feels off about requiring downstream users to declare this extension, since it's something that's needed internally.

Reply:
Users have to tell us the version of typescript to use, or point to the package.json with `ts_version_from`.
BUT it feels simpler that either
- user should just call the `npm` extension from `rules_js` and tell it to use our BUILD.typescript file
- we could put npm_typescript itself into the BCR so it's just a `bazel_dep` to get it

Reply:
Do you have a preference?

Reply:
the second one doesn't seem like it would scale, so I think we should see what the first one looks like